### PR TITLE
fix(feeds): apply alert threshold in FeedPoller

### LINKED ISSUE #472: feeds: FeedPoller ignores configured alert threshold, treats every relevant item as digest-only
## Bug

`FeedPoller` reads `config.feeds.thresholds.digest` but never reads or applies `config.feeds.thresholds.alert`. The two-tier "alert vs. digest" behavior promised by `FeedsThresholdsConfig` and the `/tune` skill is not implemented.

## Location

- Initialization: `src/distillery/feeds/poller.py:534-538`
- Threshold check during poll: `src/distillery/feeds/poller.py:854`

## What's wrong

`_poll_source` compares `adjusted_score` only against the digest threshold (default 0.60). Items with score >= 0.85 (the alert threshold) are stored identically to items at 0.61 — no flag, no separate path, no notification. The alert tier exists in config and in MCP tooling, but has no consumer.

## Why it's a bug

The `FeedsThresholdsConfig` docstring documents two distinct tiers:

- `>= alert` -> trigger an immediate alert
- `>= digest` and `< alert` -> include in next digest

`/tune` validates and writes both values, and the constraint `digest <= alert` is enforced. Users who raise/lower the alert threshold see no behavioral change because the poller never reads it. This is a feature regression / unimplemented contract, not just a config nit.

## Suggested fix

In `_poll_source`, after computing `adjusted_score`:

1. Read `self._alert_threshold` (add to `__init__`).
2. If `adjusted_score >= self._alert_threshold`, mark the entry — either via `metadata["is_alert"] = True` or by emitting through a separate alert sink.
3. Below alert but above digest, keep the current digest path.

Add an integration test that polls a synthetic high-relevance item and asserts the alert flag/sink is hit.

---
*Found by automated bug-hunt subagent.*

### PR DESCRIPTION
**Pull request by norrietaylor**
## Problem

`FeedPoller` reads `config.feeds.thresholds.digest` but never reads `config.feeds.thresholds.alert`. The two-tier behavior promised by `FeedsThresholdsConfig` (items >= `alert` trigger immediate alerts; items in `[digest, alert)` go to the digest) is therefore not implemented.

* Init was at `src/distillery/feeds/poller.py:534-538` (digest only).
* The single threshold check was at `:854` (digest vs adjusted_score).

## Fix

* Read `self._alert_threshold` from `config.feeds.thresholds.alert` in `FeedPoller.__init__`. The constraint `digest <= alert` is already enforced by `validate_config()`.
* In `_do_poll_source`, after `adjusted_score` is computed, attach `metadata["is_alert"] = True` to entries whose score meets the alert threshold. The flag is omitted (rather than set `False`) for digest-only items to match existing optional-flag conventions in the metadata dict (consistent with `metadata.backfill`).
* Defensively guard the comparison against non-numeric thresholds (e.g. `MagicMock` in some unit tests in `tests/test_feeds.py` that previously got away with mocking only `digest`) by treating any `TypeError` / `ValueError` as "not alert-worthy" — the store path must never be broken by a malformed threshold.

## Scope note (out of scope, follow-up)

A downstream alert consumer (notification sink, Slack/email push, etc.) is intentionally **not** built by this PR. A grep for `is_alert` / `alert_threshold` showed no existing consumer. The minimal contract fix is to surface the flag on the entry's metadata; wiring an actual notifier is left to a follow-up issue.

## Test plan

- [x] `pytest tests/test_poller.py::TestFeedPollerAlertThreshold -v` — 4 new regression tests:
  - `test_init_reads_alert_threshold_from_config` — init wires both thresholds
  - `test_item_at_or_above_alert_threshold_flagged` — score 0.9, alert 0.85 → `metadata.is_alert is True`
  - `test_item_in_digest_band_not_flagged_as_alert` — score 0.7 in `[digest=0.5, alert=0.85)` → no alert flag
  - `test_alert_threshold_respects_trust_weight` — trust 0.5 × score 0.9 = 0.45 < alert 0.85 → no alert flag
- [x] `pytest tests/test_poller.py tests/test_feeds.py` — 153 passed (no regressions in adjacent suites that mock config)
- [x] `ruff format src/ tests/`, `ruff check src/ tests/` — clean
- [x] `mypy --strict src/distillery/` — clean

Closes #472

## Summary by CodeRabbit

* **New Features**
  * Two-tier thresholds for feed items with an alert threshold; items meeting alert criteria are flagged and alert metadata stored with entries.

* **Bug Fixes**
  * Defensive handling for invalid alert thresholds: logs a warning once per run and disables alert tagging when threshold is unusable.

* **Tests**
  * Added tests validating alert-threshold behavior, boundary cases, and trust-weight interactions; alert threshold made configurable in tests.

### CODE REVIEW

File: src/distillery/feeds/poller.py [hunk 1]
```diff
@@ -536,6 +545,11 @@ def __init__(
             if relevance_threshold is not None
             else config.feeds.thresholds.digest
         )
+        # Two-tier threshold: items with adjusted_score >= alert threshold get
+        # flagged via metadata.is_alert (issue #472).  Items in the
+        # ``[digest, alert)`` band remain digest-eligible only.  The constraint
+        # ``digest <= alert`` is enforced by validate_config().
+        self._alert_threshold = config.feeds.thresholds.alert
         self._reader = reader
```
[the remark this answers is not in the document]
Comment (norrietaylor) [reply src/distillery/feeds/poller.py:556]: Applied in 6116888 — wrapped the float coercion in a try/except and emit a single WARNING per poller instance via a `_alert_threshold_warning_emitted` one-shot flag, so misconfiguration is observable without per-item log spam. There is only one coercion site in this module (the same site referenced as "Also applies to: 882-889"), so no second occurrence to patch.

**Comment by norrietaylor** [comment]
@coderabbitai review — the original concern was addressed in commit 6116888 and the inline thread is resolved. Could you take another pass and update the review state if it looks good?

**Comment by norrietaylor** [comment]
@coderabbit review
